src/ucp/core/ucp_context.c

```diff
@@ -544,7 +546,7 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "in addition to what is set explicitly by the user. \n"
    "Possible values are: no_imm_cmpl, fast_cmpl, force_imm_cmpl, multi_send.",
    ucs_offsetof(ucp_context_config_t, extra_op_attr_flags),
-   UCS_CONFIG_TYPE_BITMAP(ucp_extra_op_attr_flags_names)},
+   UCS_CONFIG_TYPE_FLAGS((const void *)&ucp_extra_op_attr_flags_args)},
```

Comment:
pls fix comment on line 547, as `no_imm_cmpl`  and `force_imm_cmpl`  are not the options anymore.